### ISSUE TITLE
search() example in docs doesn't work

### ISSUE THREAD

**Issue by erikrose** [pyelasticsearch/pyelasticsearch#22]
The `conn.search(query)` example in the readme doesn't seem to work:

```
conn.search({ "query_string": { "query": "name:tester" }, "filtered": { "filter": { "range": { "age": { "from": 27, "to": 37\
```

DEBUG 2012-08-30 22:41:39,938 connectionpool "GET /_search?q=%7B%27query_string%27%3A+%7B%27query%27%3A+%27name%3Atester%27%7D%2C+%27filtered%27%3A+%7B%27filter%27%3A+%7B%27range%27%3A+%7B%27age%27%3A+%7B%27to%27%3A+37%2C+%27from%27%3A+27%7D%7D%7D%7D%7D HTTP/1.1" 500 64354
**\* ElasticSearchError: Non-OK status code returned (500) containing u'SearchPhaseExecutionException[Failed to execute phase [query], total failure; shardFailures {[dH2uhViYRnyupZTQrGM0IQ][api_small][16]: ...

In short, it tried to stick the query in a querystring arg rather than in the request body. Maybe we should change the example to use the `body` kwarg. At any rate, I got my code working, but I thought I'd mention it.
